test/compliance/test_kaldi.py

```diff
@@ -172,7 +177,7 @@ def get_output_fn(sound, args):
                 window_type=args[12])
             return output
 
-        self._compliance_test_helper('spec', 131, 13, get_output_fn)
+        self._compliance_test_helper(self.test_filepath, 'spec', 131, 13, get_output_fn, atol=1e-3)
```

Comment:
Why are you decreasing the tolerance on unrelated tests?

Reply:
You mean the rtol part? spec was previously atol=1e-3 rtol=0
https://github.com/pytorch/audio/pull/119/files#diff-482e7b29706be7db3f81fe0095e058c7R136

I'm just separating the accuracy parameters for the various features (fbank, spec, etc.)